src/Marten.Testing/Acceptance/foreign_keys.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using Marten.Testing.Documents;
+using Shouldly;
+using Xunit;
+
+namespace Marten.Testing.Acceptance
+{
+    public class foreign_keys: IntegratedFixture
+    {
+        [Fact]
+        public void can_insert_document_with_null_value_of_foregin_key()
```

Comment:
@oskardudycz Please fix typo in the method name `foregin` to `foreign`? Looks like this it is there in all the unit test method names.

Reply:
@mysticmind fixed. There is nothing better than copy/pasting mistakes... 😄